tools/sharkbait/src/sharkbait/permissions.clj

```diff
@@ -21,9 +22,10 @@
   "Sets the allowed actions for a permission definition."
   [permission-def]
   (let [delegate (.getAttributeDefActionDelegate permission-def)]
-    (.configureActionList delegate ["read" "write" "own"])
-    (add-implied-action delegate "own" "write")
-    (add-implied-action delegate "write" "read"))
+    (.configureActionList delegate [read write own originator])
+    (add-implied-action delegate originator own)
```

Comment:
Alright, makes sense. I think "originator" might be what's confusing me, because it sounds like a provenance term which you'd expect to remain valid even after it's made public (which this can't be, given the inheritance). So I wonder if there's maybe a better name -- I'm still not completely sure what the user-facing role of this level would be, which would determine that I think, but something like "lead" might make sense if the goal is just something like "ownership, but can't be revoked by other owners". Given you know what your plans are, though, I'm sure you have better ideas, heh